managed/services/inventory/services.go

```diff
@@ -76,6 +76,40 @@ func (ss *ServicesService) List(ctx context.Context, filters models.ServiceFilte
 	return res, nil
 }
 
+// ListActiveServiceTypes lists all active Service Types
+//
+//nolint:unparam
+func (ss *ServicesService) ListActiveServiceTypes(ctx context.Context) ([]inventorypb.ServiceType, error) {
+	var types []models.ServiceType
+	err := ss.db.InTransaction(func(tx *reform.TX) error {
+		var err error
+		types, err = models.FindActiveServiceTypes(tx.Querier)
+		return err
+	})
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Let's keep it consistent with other parts
```suggestion
	e := ss.db.InTransaction(func(tx *reform.TX) error {
		var err error
		types, err = models.FindActiveServiceTypes(tx.Querier)
		return err
	})
	if e != nil {
		return nil, e
	}
```